src/System.Text.Json/src/System/Text/Json/Document/JsonProperty.cs

```diff
@@ -24,6 +26,69 @@ internal JsonProperty(JsonElement value)
         /// </summary>
         public string Name => Value.GetPropertyName();
 
+        /// <summary>
+        /// Compares the string text to the unescaped JSON token value in the source and returns true if they match.
+        /// </summary>
+        /// <param name="text">The text to compare against.</param>
+        /// <returns>True if the JSON token value in the source matches the look up text.</returns>
```

Comment:
```suggestion
        ///   <see langword="true" /> if the name of this property matches <paramref name="text"/>,
        ///   <see langword="false" /> otherwise.
```